integration-libs/cdc/organization/administration/cdc-org-unit.service.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { OrgUnitService } from '@spartacus/organization/administration/core';
+@Injectable({
+  providedIn: 'root',
+})
+export class CdcOrgUnitService extends OrgUnitService {
+  isUpdatingUnitAllowed(): boolean {
```

Comment:
Recommended to make it `protected`

Reply:
if we make 'isUpdatingUnitAllowed' protected , it will be accessible only within class 'OrgUnitService' and its subclasses. We wont be able to call it from unit-details.component.ts. Hence not making the change.

Reply:
This would allow other components to change the behaviour and thus update the units, even if we disable it.